code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -104,6 +104,14 @@
 	amount_per_transfer_from_this = 20
 	list_reagents = list("ephedrine" = 10, "coffee" = 10)
 
+/obj/item/weapon/reagent_containers/hypospray/medipen/leppen //basilisks were buffed

```

Comment:
I actually haven't died to Basilisks ever, but the recent buff makes it nearly impossible to fight en-masse.

Before the buff, you could easily kill basilisks, but the buff changed it so it's nearly impossible due to the incredibly delayed temperature burn damage.